YIF.Core.Service/Concrete/Services/SuperAdminService.cs

```diff
@@ -242,7 +242,7 @@ public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteInstituti
         public async Task<ResponseApiModel<DescriptionResponseApiModel>> DisableInstitutionOfEducationAdmin(string adminId)
         {
             var result = new ResponseApiModel<DescriptionResponseApiModel>();
-            var ch = await _institutionOfEducationAdminRepository.GetByUserId(adminId);
+            var ch = await _institutionOfEducationAdminRepository.GetUserByAdminId(adminId);
```

Comment:
when you changed such line unit tests for such method dropped, that's why you should also fix unit tests

Reply:
Unit test Fixed